app2/src/main/java/com/example/recipebox/MainActivity.kt

```diff
@@ -98,28 +99,43 @@ fun Scaffold() {
 				}
 			)}
 
-	) { padding ->
-			Column(modifier = Modifier.padding(10.dp)) {
+	) {
+		Column(modifier = Modifier.padding(10.dp)) {
 				OutlinedTextField(
 					value = recipeName,
 					onValueChange = { recipeName = it },
 					label = { Text(stringResource(R.string.recipeName)) },
-					modifier = Modifier.fillMaxWidth()
+					modifier = Modifier.fillMaxWidth(),
 				)
 				Button(
 					onClick = {
-						Toast.makeText(context, "$recipeName", Toast.LENGTH_LONG).show()
```

Comment:
This was redundant so I just replaced it with the val.

Reply:
Yeah.  LOL.